build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -13,6 +13,17 @@
                            Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
         <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
       </NupkgsForPublishing>
+
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.$(Architecture).*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' " />
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " />
+
+      <NupkgsForPublishing Include="@(VSNupkgsForPublishing)">
+        <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+      </NupkgsForPublishing>
```

Comment:
@wli3 @dsplaisted Any reason these changes were not in 2.1.8xx?

Reply:
Perhaps because we don't insert 2.1.8xx into VS at all.

Reply:
Ahh. No harm in this change though. Just makes it go to the blob feed.